partners.yml

```diff
@@ -529,21 +541,80 @@ components:
       required:
       - status
       - message
+      - errors
       properties:
         status:
           type: integer
           description: HTTP status code
-          example: 400
         message:
           type: string
           description: Main error reason
-          example: Invalid Data
         errors:
           type: array
           items:
             type: string
             description: Optional list of errors
-            example: Field ... should be defined
+
+    dataError:
+      allOf:
+        - $ref: "#/components/schemas/error"
+        - type: object
+          properties:
+            status:
+              example: 400
+            message:
+              example: INVALID_REQUEST
+            errors:
+              type: array
+              items:
+                example: Field ... should be defined
+
+    authError:
+      allOf:
+        - $ref: "#/components/schemas/error"
+        - type: object
+          properties:
+            status:
+              example: 401
+            message:
+              example: Unauthorized
+
+    serverError:
+      allOf:
+        - $ref: "#/components/schemas/error"
+        - type: object
+          properties:
+            status:
+              example: 500
+            message:
+              example: Internal Server Error
+
+    serviceError:
+      allOf:
+        - $ref: "#/components/schemas/error"
+        - type: object
+          properties:
+            status:
+              example: 503
+            message:
+              example: Service unavailable
+
+    sendCodeError:
+      allOf:
+        - $ref: "#/components/schemas/error"
+        - type: object
+          properties:
+            status:
+              example: 400
+            message:
+              type: string
+              description: INVALID_REQUEST mean that the payload of the request is invalid, check the errors list to get more information. INVALID_TOKEN mean that the code passed as parameter was not recognised by our authentication partner, the end user should do the authentication flow again to obtain a new token. INVALID_IDENTITY means that the bank account retrieved using the code does not belong to the socialSecurityNumber, the end user should do the authentication flow again with a different bank where he/she has an account in his/her name. INVALID_ACCOUNTS means that the end user does not have any salary account at the selected bank and needs to redo the authentication flow selecting another bank where he/she has a salary account.
```

Comment:
`INVALID_REQUEST mean that` isn't supposed to be `INVALID_REQUEST means that`? 🤔